demo/src/main.tsx

```diff
@@ -40,7 +40,7 @@ const realtimeClient = new Ably.Realtime({
   clientId,
 });
 
-const chatClient = new ChatClient(realtimeClient, { logLevel: LogLevel.Debug });
+const chatClient = new ChatClient(realtimeClient, { logLevel: LogLevel.Debug, logHandler: console.log });
```

Comment:
FWIW the default is to use an inbuilt logger based on console.log, which also includes properly using `console.error` in error situations.

Should we let it use the default here?